### ISSUE TITLE
Django Workshop

### ISSUE THREAD

**Issue by RajuKoushik** [iiitv/workshops#11]
What about a Django workshop ?

**Comment by aviaryan** [comment]
Good idea. Please assign people who you think can give a Django workshop.

**Comment by aviaryan** [comment]
In case you can't assign people, just mention them here.

**Comment by RajuKoushik** [comment]
Assign me , @aviaryan and @singhpratyush !!

**Comment by aviaryan** [comment]
I haven't worked with Django in a long time so unfortunately I am not suitable to give a session on it.

**Comment by arpit1997** [comment]
Which batches will be the target audiences?

**Comment by yashLadha** [comment]
Mainly the second year one? Because still needs to build knowledge of data structure and algorithm.
